data/method_process.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# imports
+import os
+import sys
+from pathlib import Path
+from typing import List, Optional
+import typer
+import pandas as pd
+import numpy as np
+from typing import Tuple
+
+# TODO import helpers and file class
+from postprocessing_variant_calls.maf.helper import (
+    check_maf,
+    check_txt,
+    check_separator,
+    read_tsv,
+    MAFFile,
+    gen_id_tsv,
+)
+
+
+# TODO add help and sub-command name
+app = typer.Typer(help="post-processing command tagging maf files")
+
+
+@app.command(
+    "common_variant",
+    help="Tag a variant in a MAF file as common variant based on GNOMAD AF",
+)
+# TODO update sub-command function name
+def common_variant(
+    maf: Path = typer.Option(
+        ...,
+        "--maf",
+        "-m",
+        exists=True,
+        file_okay=True,
+        dir_okay=False,
+        writable=False,
+        readable=True,
+        resolve_path=True,
+        help="MAF file to subset",
+    ),
+    output_maf: Path = typer.Option(
+        "output.maf", "--output", "-o", help="Maf output file name."
+    ),
+    separator: str = typer.Option(
+        "tsv",
+        "--separator",
+        "-sep",
+        help="Specify a seperator for delimited data.",
+        callback=check_separator,
+    ),
+):
+    # prep maf
+    mafa = MAFFile(maf, separator)
+    mafa = mafa.tag("common_variant")
+    mafa.to_csv(f"{output_maf}".format(outputFile=output_maf), index=False, sep="\t")
+    return 0
```

Comment:
Not a review comment but a question-- I noticed you return "0" at the end of your sub-command functions. What is the purpose of returning 0 as compared to a string value indicating the execution of the sub-command has completed or not returning anything at all?

Reply:
In python, if I'm writing a file in a function as opposed to returning something, I still like to return a 0 to indicate a successful run of the function. Some languages like C require you return something at the completion of a function. It's not required in python, but something I think is decent style. We can change the return as well.